src/app/loadout-builder/LockArmorAndPerks.tsx

```diff
@@ -34,16 +34,15 @@ import { D2ManifestDefinitions } from 'app/destiny2/d2-definitions';
 import { settingsSelector } from 'app/settings/reducer';
 import LockedArmor2ModIcon from './LockedArmor2ModIcon';
 import ModPicker from './ModPicker';
+import { LoadoutBuilderAction } from './LoadoutBuilder';
 
 interface ProvidedProps {
   selectedStore: DimStore;
   items: ItemsByBucket;
   lockedMap: LockedMap;
   lockedSeasonalMods: LockedModBase[];
   lockedArmor2Mods: LockedArmor2ModMap;
-  onLockedMapChanged(lockedMap: ProvidedProps['lockedMap']): void;
-  onSeasonalModsChanged(mod: LockedModBase[]): void;
-  onArmor2ModsChanged(mods: LockedArmor2ModMap): void;
+  lbDispatch: Dispatch<LoadoutBuilderAction>;
```

Comment:
This is cool!

Reply:
Yeah I really like just passing the dispatch around! It's much cleaner.